staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -107,3 +107,12 @@ if [ "${GENS}" = "all" ] || grep -qw "informer" <<<"${GENS}"; then
            --output-package ${OUTPUT_PKG}/informers \
            "$@"
 fi
+
+if [ "${GENS}" = "all" ] || grep -qw "openapi" <<<"${GENS}"; then
+  echo "Generating OpenAPI definitions for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/openapi"
+  ${GOPATH}/bin/openapi-gen \
+           --input-dirs $(codegen::join , "${EXT_FQ_APIS[@]}") \
+           --input-dirs "k8s.io/api/core/v1,k8s.io/apimachinery/pkg/api/resource,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/version,k8s.io/apimachinery/pkg/runtime" \
```

Comment:
this is kubernetes intedependent. There shouldn't be references to other kubernetes packages.

Reply:
`k8s.io/api/core/v1`? 
You require `k8s.io/apimachinery` for sure - `ObjectMeta` etc...

Reply:
apimachinery is ok, core/v1 is not.